### DISCUSSION TITLE
Support for new Google Identity Services provider

### DISCUSSION THREAD

**Discussion by anargiris** [nextauthjs/next-auth#4644] [Ideas]
### Description 📓

Hello everyone and thanks for your amazing work on this library!

I wanted to ask if there is new support coming for the Google Identity Services authentication, since Google will retire their Google sign-in platform. More information here:

https://developers.google.com/identity

Thanks a lot!

### How to reproduce ☕️

No reproduce since this is a feature request.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
There is a user-land tutorial implementing Google One Tap: https://github.com/nextauthjs/next-auth/issues/1058
